src/test/resources/features/Local/01_ProfileSetupAndMaintain.feature

```diff
@@ -226,5 +226,7 @@ Feature: Trader Goods Profile setup and Previous movement records journey
     And I select continue
     Then I should be on the 'Profile details' page
     And My 'NIPHL Question' has the value No
+    When I select the back link
```

Comment:
Can we use "When I click on the Back to homepage link" as it is more relevant to this ticket